src/components/GameView/MoveChoiceOverlay.vue

```diff
@@ -131,12 +131,27 @@ export default {
       };
     },
     oneOffMove() {
+      let oneOffDisabled = this.allMovesAreDisabled;
+      let oneOffDisabledExplanation = this.disabledText;
+      //Check deck while playing 7s
+      if (this.selectedCard.rank === 7) {
+        if (!this.$store.state.game.topCard) {
+          oneOffDisabled = true;
+          oneOffDisabledExplanation = 'Cannot play one-off with an empty deck';
+        } else if (
+          this.selectedCard.id === this.$store.state.game.topCard.id &&
+          !this.store.state.game.secondCard
```

Comment:
I don't think this is currently covered by your test cases. Instead the first chunk of logic around the missing topCard looks to be tested twice. Let's rework the 2nd test case to cover this situation